Makefile

```diff
@@ -598,7 +598,7 @@ ifdef PIP_VERSION
 PIP_THEMIS_INSTALL := $(shell pip freeze |grep themis)
 endif
 
-pythemis_install: CMD = cd src/wrappers/themis/python/ && python3 setup.py install --record files3.txt
+pythemis_install: CMD = cd src/wrappers/themis/python/ && pip3 install .
```

Comment:
Hmm, it requires additional testing, preferably not manual as I did. But I guess for now we can add this command just to be sure.

Reply:
I'm thinking, maybe we should add that command to the `prepare_tests_all`? Updating pip each time pythemis_install is called doesn't sound good to me. For example, maybe there are reasons why users don't want to update pip.

As I understand, only we use `prepare_tests_all` in our testing environments, so putting this command there should prevent problems with pip on our test machines. At the same time, if users, who install pythemis from the sources, will have to fix their environments in controlled way, if something goes wrong.

Reply:
agree, better if we will add it in our own test environment. but starting installation via pip we should be sure that this approach supported by all supported OS, because approach with setup.py works for all of them

Reply:
Hmmm, another thought came to mind. The new pip versions (for example, in debian bookworm) don't want to install packages globally, so we will have to fix something (use virtual environments, pipx, etc). So, maybe it is better to postpone the change and fix only if it breaks?

Currently, it seems like a small change but it requires extensive testing on all our platforms.